tests/acceptance/init-test.js

```diff
@@ -166,4 +166,67 @@ describe('Acceptance: ember init', function () {
+  // -------------------------------
+  // Good: Default
+  it('ember init without --lang flag (default) has no lang attribute in index.html', async function () {
+    await ember(['init', '--skip-npm', '--skip-bower', '--skip-git']);
+    confirmBlueprinted();
+    // expect(file('app/index.html')).to.contain('<html>');
+  });
+
+  // Good: Correct Usage
+  it('ember init with --lang flag and valid code assigns lang attribute in index.html', async function () {
+    await ember(['init', '--skip-npm', '--skip-bower', '--skip-git', '--lang=en-US']);
+    // expect(file('app/index.html')).to.contain('<html lang="en-US">');
+  });
+
+  // Misuse: possibly an attempt to set app programming language
+  it('ember init with --lang flag and programming language fails with an error message', async function () {
+    let err = await expect(ember(['init', '--skip-npm', '--skip-bower', '--skip-git', '--lang=typescript'])).to.be
+      .rejected;
+    expect(err.name).to.equal('SilentError');
+    expect(err.message).to.be.ok;
+    expect(err.message).to.include('An error with the `--lang` flag returned the following message:');
+    expect(err.message).to.include('Information about using the `--lang` flag:');
+    expect(err.message).to.include('set the app programming language');
+    expect(err.message).to.include('typescript');
+    expect(err.message).to.not.include('ember-cli command option');
+  });
+
+  // Misuse: No specification + declared option
+  it('ember init with --lang flag but no specification fails with an error message; absorbs ember-cli option (declared)', async function () {
+    let err = await expect(ember(['init', '--skip-npm', '--skip-bower', '--lang', '--skip-git'])).to.be.rejected;
+    expect(err.name).to.equal('SilentError');
+    expect(err.message).to.be.ok;
+    expect(err.message).to.include('An error with the `--lang` flag returned the following message:');
+    expect(err.message).to.include('Information about using the `--lang` flag:');
+    expect(err.message).to.not.include('set the app programming language');
+    expect(err.message).to.include('ember-cli command option');
+    expect(err.message).to.include('--skip-git');
+  });
+
+  // Misuse: No specification + hidden option
+  it('ember init with --lang flag but no specification fails with an error message; absorbs appended ember-cli option (hidden)', async function () {
+    let err = await expect(ember(['init', '--skip-npm', '--skip-bower', '--skip-git', '--lang'])).to.be.rejected;
+    expect(err.name).to.equal('SilentError');
+    expect(err.message).to.be.ok;
+    expect(err.message).to.include('An error with the `--lang` flag returned the following message:');
+    expect(err.message).to.include('Information about using the `--lang` flag:');
+    expect(err.message).to.not.include('set the app programming language');
+    expect(err.message).to.include('ember-cli command option');
+    expect(err.message).to.include('--disable-analytics');
+  });
+
```

Comment:
I think we should remove these tests, our normal command option parsing should enforce this. If it doesn't, please make a (somewhat) detailed issue and then we can track that down after landing this.
 
```suggestion
```

Reply:
Removed when consolidating testing suite to acceptance/new-test in 72e10e9 👍